apps/example/public/mockServiceWorker.js

```diff
@@ -0,0 +1,292 @@
+/* eslint-disable */
+/* tslint:disable */
+
+/**
+ * Mock Service Worker.
+ * @see https://github.com/mswjs/msw
+ * - Please do NOT modify this file.
+ * - Please do NOT serve this file on production.
+ */
+
+const PACKAGE_VERSION = "2.6.6"
```

Comment:
Do we really need this whole logic?

Reply:
I see your point but this file is actually auto-generated by `msw` as part of the setup and ideally should not to be changed.

https://mswjs.io/docs/integrations/browser